Clustering/hybridEcalSplitter/readme.txt

```diff
@@ -1,3 +1,25 @@
+DDStripSplitter
+---------------
+
+implementation of Kotera's strip split method, for new DD4hep based models
+[hybrid case ( mix of scint and si ) not yet implemented]
+
+Daniel Jeans, Nov 2018.
+
+example steering snippet:
+
+  <processor name="myDDStripSplitter" type="DDStripSplitter">
+    <parameter name="ECALcollections_tranStrips"> ECalBarrelScOddCollectionRec ECalEndcapScOddCollectionRec </parameter>
+    <parameter name="ECALcollections_longStrips"> ECalBarrelScEvenCollectionRec ECalEndcapScEvenCollectionRec </parameter>
+    <parameter name="cellSize0"> 5. </parameter>
```

Comment:
You should get this info from DD4hep segmentation I believe

Reply:
@rete thanks for the detailed QA. done.